### PR TITLE
DOCS-2509: Update README.md

### PR DESCRIPTION
**Pull request by sguequierre**
* updating based on old rdk docs 
* support can be updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,130 @@
-# viam-tflite-cpu
-the module for running tflite models on the cpu
+# [`tflite_cpu` module](https://app.viam.com/module/viam/tflite_cpu)
+
+This [module](https://docs.viam.com/registry/modular-resources/) implements the [`rdk:service:mlmodel` API](https://docs.viam.com/appendix/apis/services/ml/) in an `tflite_cpu` model.
+With this model, you can deploy [TensorFlow Lite](https://www.tensorflow.org/lite) model files as long as your models adhere to the [model requirements](#model-requirements).
+<!-- It is supported on any CPU and Linux, Raspbian, MacOS and Android machines. -->
+It is supported on any CPU and Linux and MacOS machines.
```
Comment (sguequierre) [inline README.md]:
```suggestion
It is supported on any CPU and Linux, Raspbian and MacOS machines.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,130 @@
-# viam-tflite-cpu
-the module for running tflite models on the cpu
+# [`tflite_cpu` module](https://app.viam.com/module/viam/tflite_cpu)
+
+This [module](https://docs.viam.com/registry/modular-resources/) implements the [`rdk:service:mlmodel` API](https://docs.viam.com/appendix/apis/services/ml/) in an `tflite_cpu` model.
+With this model, you can deploy [TensorFlow Lite](https://www.tensorflow.org/lite) model files as long as your models adhere to the [model requirements](#model-requirements).
+<!-- It is supported on any CPU and Linux, Raspbian, MacOS and Android machines. -->
+It is supported on any CPU and Linux, Raspbian and MacOS machines.
```
Comment (sguequierre) [inline README.md:6]: @bhaney can you confirm that this is supported on raspbian machines and not yet supported on android?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,130 @@
...
+- a model available on your machine
+
+## Configure your tflite_cpu ML model
+
+Navigate to the [**CONFIGURE** tab](https://docs.viam.com/configure/) of your [machine](https://docs.viam.com/fleet/machines/) in the [Viam app](https://app.viam.com/).
+[Add `ML model / TFLite CPU` to your machine](https://docs.viam.com/configure/#services).
+Enter a name or use the suggested name for your service.
+Click **Add module** and click **Create**.
+
+If you choose to use **Builder** mode in the Viam app you can choose to configure your service with an existing model on the machine or deploy a model onto your machine:
```
Comment (npentrel) [inline README.md]: You can also do that with JSON mode, just harder
```suggestion
You can choose to configure your service with an existing model on the machine or deploy a model onto your machine:
```

**Review by npentrel — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (sguequierre) [reply README.md]: true

File: README.md [hunk 4]
```diff
@@ -1,2 +1,130 @@
...
+- a model available on your machine
+
+## Configure your tflite_cpu ML model
+
+Navigate to the [**CONFIGURE** tab](https://docs.viam.com/configure/) of your [machine](https://docs.viam.com/fleet/machines/) in the [Viam app](https://app.viam.com/).
+[Add `ML model / TFLite CPU` to your machine](https://docs.viam.com/configure/#services).
+Enter a name or use the suggested name for your service.
+Click **Add module** and click **Create**.
+
+You can choose to configure your service with an existing model on the machine or deploy a model onto your machine:
```
Comment (sguequierre) [inline README.md]:
```suggestion
You can choose to configure your service with an existing model on the machine or deploy a model onto your machine. 
To do so in **BUILDER** mode on the Viam app:
```

File: README.md (hunk 2 above)
Comment (bhaney) [reply README.md:6]: correct, not yet on android, but yes on all pi's

**Pull request merged**